File: Javascript/backend_nest/src/infra/db/inmemory/entities/UserInMemory.ts
export class UserInMemory {
  id: number;
  email: string;
  role: string;
  firstName: string;
  lastName: string;

  constructor(id: number, email: string, role: string, firstName: string, lastName: string) {
    this.id = id;
    this.email = email;
    this.role = role;
    this.firstName = firstName;
    this.lastName = lastName;
  }
}

File: Python/backend_flask_archi_feature/user/db/user_inmemory.py
class UserInMemory:
    def __init__(self, id: int, email: str, password: str, role: str, first_name: str, last_name: str):
        self.id = id
        self.email = email
        self.password = password
        self.role = role 
        self.first_name = first_name
        self.last_name = last_name